account_vat_period_end_statement/readme/DESCRIPTION.rst

```diff
@@ -9,6 +9,11 @@
     (vedere Configurazione per generare correttamente i periodi).
     I debiti e crediti precedenti vengono caricati dalle liquidazioni IVA precedenti, in base allo stato del loro pagamento.
 
+    A seguito dell’introduzione della normativa OSS, per chi fa vendite online su diversi paesi, deve necessariamente escludere dalla liquidazione iva ciò che farà parte della liquidazione OSS.
+    E’ stato introdotto un nuovo campo per filtrare le imposte da elaborare, quindi va indicato il conto configurato nelle imposte  “Conto utilizzato per la liquidazione IVA”.
+    Quindi selezionare il conto IVA debito per elaborare liquidazione iva su tutte le imposte che hanno in configurazione il conto IVA debito.  Per la liquidazione OSS invece selezionare conto  dedicato ad esempio IVA debito OSS Francia.
+    E’ necessario creare un periodo di dichiarazione IVA dedicato all’OSS .
```

Comment:
Come mai?

Reply:
Chiaro, ma a cosa serve un periodo dedicato? Fai 2 dichiarazioni e filtri le imposte in base a `account_ids`

Reply:
Mmm...sbaglio o non si possono avere 2 dichiarioni con la stessa data?

Reply:
Ah ho capito, `date_range_ids` è un o2m, quindi lo stesso periodo non può essere associato a dichiarazioni diverse.

Potrebbe essere considerato un limite della progettazione iniziale?
Cioè, se `date_range_ids` fosse un m2m, con dei controlli per evitare all'utente di fare errori, funzionerebbe meglio?
Vale la pena valutare il cambio di struttura dati? Con relativo script di migrazione ovviamente

Reply:
Il prototipo qui https://github.com/OCA/l10n-italy/pull/2490
@jado95 che ne pensi?

Reply:
Ti ho anche creato https://github.com/jado95/l10n-italy/pull/6 , giusto per tenerne traccia